editor/src/clj/editor/code/view.clj

```diff
@@ -1783,9 +1784,12 @@
   (init-property-and-bind-preference! find-case-sensitive-property prefs "code-editor-find-case-sensitive" false)
   (init-property-and-bind-preference! find-wrap-property prefs "code-editor-find-wrap" true)
   (init-property-and-bind-preference! font-size-property prefs "code-editor-font-size" default-font-size)
+  (init-property-and-bind-preference! font-name-property prefs "code-editor-font-name" "")
   (init-property-and-bind-preference! visible-indentation-guides-property prefs "code-editor-visible-indentation-guides" true)
   (init-property-and-bind-preference! visible-minimap-property prefs "code-editor-visible-minimap" true)
-  (init-property-and-bind-preference! visible-whitespace-property prefs "code-editor-visible-whitespace" true))
+  (init-property-and-bind-preference! visible-whitespace-property prefs "code-editor-visible-whitespace" true)
+  (when (clojure.string/blank? (.getValue font-name-property))
```

Comment:
If the user has set a blank string in the prefs_dialog we revert back to "Dejavu Sans Mono"

Reply:
It is a single text field?
Would it be possible to select a font file instead? It feels like it's very easy to misspell the font name?

Reply:
There is no file selector functionality in the Preference dialog unfortunately. It was the same when we got a PR for additional Keymaps. I'll merge it as it is and perhaps we can add that later.